index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <content>
+  <body id="body">
+
+    <ul id="FirstAndPike">
+      <h1> FirstAndPike </h1>
```

Comment:
I would avoid putting tags other than h1s inside of a ul. Maybe you could wrap the ul in a div or section, and then have the h1 as a sibling of the ul?